### DISCUSSION TITLE
dismissing clarity wizard when clicking on the background overlay

### DISCUSSION THREAD

**Discussion by RROSANN** [vmware/clarity#6026] [Q&A]
According to the documentation there is a possibility to overwrite them but I did not found it anywhere, does anyone know what is the binding to that function? 

Below is the statement I found on the main documentation...

Clarity Wizards are not dismissed when clicking on the background overlay. This prevents losing information or data. An option to override this is available.

https://clarity.design/angular-components/wizard/#dismissing-wizard

**Comment by Jinnie** [comment]
Looking at the code, we don't have such an option. Though it may be achievable programatically. Will need to update the documentation.
